wsd/Storage.cpp

```diff
@@ -74,15 +74,44 @@ bool StorageBase::SSLEnabled = false;
 
 std::string StorageBase::getLocalRootPath() const
 {
+    std::string localStorePath(_localStorePath);
     std::string localPath = _jailPath;
     if (localPath[0] == '/')
     {
         // Remove the leading /
         localPath.erase(0, 1);
     }
 
+    // Use where mountJail of kit/Kit.cpp mounts /tmp for this path *from* rather than
+    // where it is mounted *to*, so this process doesn't need the mount visible to it
+    if (COOLWSD::EnableMountNamespaces && !localPath.empty())
```

Comment:
We should prolly do this un-conditionally in the next red release =)

Reply:
yeah, needs checking for existing fallback non-mount cases where I think we use link as to what links we are using and perhaps need to align the link fallback with the mount case so its always the same.